src/main/kotlin/hs/kr/equus/user/domain/auth/service/PassPopupService.kt

```diff
@@ -0,0 +1,106 @@
+package hs.kr.equus.user.domain.auth.service
+
+import hs.kr.equus.user.domain.auth.presentation.dto.request.PassPopupRequest
+import hs.kr.equus.user.global.exception.InternalServerErrorException
+import hs.kr.equus.user.global.utils.pass.RedirectUrlChecker
+import kcb.module.v3.OkCert
+import org.json.JSONObject
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class PassPopupService(
+    private val redirectUrlChecker: RedirectUrlChecker
+) {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line